openmc/lib/cell.py

```diff
@@ -101,7 +103,7 @@ class Cell(_FortranObjectWithID):
         Name of the cell
     num_instances : int
         Number of unique cell instances
-    bounding_box : 2-tuple of numpy.ndarray
+    bounding_box : openmc.BoundingBox
```

Comment:
The description of this property in the line below is a little off now.

Reply:
thanks Patrick, I have changed the docstring to match the equivalent docstring in cell.py